src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/custom-anomalies.mdx

```diff
@@ -0,0 +1,104 @@
+* Set whether you want the condition to check for upper violations, lower violations, or both.
+* Set the behavior that will trigger an incident (for example: "deviating for more than five minutes") 
+* Adjust how sensitive the condition is to fluctuations in the data source by adjusting the sliding bar to more violations or fewer violations. This bar calculates how many standard deviations away from the prediction  you want to be notified.
+
+<img
+  title="Custom anomaly configuration"
+  alt="A screenshot demonstrating how to configure custom anomalies"
+  src={alertsConfigureYourAnomalyThresholds}
+/>
+
+When your data escapes the predicted "normal" behavior and meets the criteria you've chosen, you'll receive a notification.
+
+Here are some tips for setting anomaly thresholds:
+
+* Set the [anomaly direction](#upper-lower) to monitor violations that happen either above or below the anomaly.
+* Set the preview chart to either [2 days or 7 days](#days) of displayed data. (Not applicable for [NRQL alert conditions](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries).)
+* Use the slider bar to adjust the <img title="Alerts v3: Critical alert icon" alt="icon-alert-critical.png" src={iconAlertCritical}/> **Critical** threshold sensitivity, represented in the preview chart by the light gray area around the anomaly. The tighter the band around the anomaly, the more sensitive it is and the more violations it will generate.
+* Optional: You can create a <img title="Alerts v3: Warning alert icon" alt="icon-alert-warning.png" src={iconAlertWarning}/> [**Warning** threshold](/docs/alerts/new-relic-alerts/defining-conditions/set-thresholds-alert-condition#threshold-levels) (the darker gray area around the anomaly).
+* For NRQL alerts, see the [allowed types of NRQL queries](/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries#threshold-types).
+* If the alert condition applies to multiple apps, you can select a choice from the dropdown above the chart to use different metrics. (Not applicable for NRQL alert conditions.)
+
+### Faceted anomaly conditions [#anomaly-facets]
+
+Once you've run a faceted NRQL query for a anomaly condition, you can scope your results to a single anomaly.
+
+<img
+  title="Faceted anomalies"
+  alt="A screenshot depicting faceted anomalies"
+  src={alertsFacetedAnomalies}
+/>
+
+The single time series shows the anomaly, threshold band, and one or more violation regions, if there are any.
+
+<img
+  title="Configured faceted anomalies"
+  alt="A screenshot depicting a deeper dive into facted anomalies "
+  src={alertsFacetedAnomaliesTwo}
+/>
+
+### Anomaly direction: select upper or lower ranges
+
+You can choose whether you want the condition to violate for behavior that goes above the predicted value ("upper") or that goes below the predicted value ("lower"), or that goes either above or below. You choose these with the  prediction direction selector.
```

Comment:
"The condition to violate for" reads a bit strange here. Also there is a double space before prediction in the last sentence.

Reply:
done